### DISCUSSION TITLE
Missing important feature "WhatsApp Embedded Signup"

### DISCUSSION THREAD

**Discussion by Abodolla** [chatwoot/chatwoot#10421] [Feature Requests]
Description:

The Embedded Signup feature for WhatsApp API is the ideal and officially recommended method by Meta. This feature simplifies the process of registering phone numbers and enables users to integrate with the WhatsApp API quickly and seamlessly.

<img width="274" alt="Connect_Account_WhatsApp" src="https://github.com/user-attachments/assets/283bdea8-2599-43b2-b22a-b98205a3c67b">

**Comment by Abodolla** [comment]
https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/facebook-channel-setup

like this but for WhatsApp.

**Comment by FacundoSuarez-46** [comment]
Hi, we have developed this feat at https://nuuxia.com/

When the embedded signup is finished the new WA inbox is created

<img width="1409" alt="Screenshot 2024-12-30 at 5 31 14 PM" src="https://github.com/user-attachments/assets/2e122c94-296b-495f-b232-77ee88e1524d" />

**Reply by pranavrajs** [reply]
@Abodolla Would you like to contribute?

**Reply by Abodolla** [reply]
Yes, I'd be happy to contribute

**Reply by stenioanibal** [reply]
Hey folks. Did you have any news about this? Really appreciate!

**Reply by badereddineqodia** [reply]
@pranavrajs i can contibute as well !

**Reply by badereddineqodia** [reply]
@Abodolla would you if i have your contact and talk about it in order to do our best to contribute

**Reply by Abodolla** [reply]
I’ve done it successfully!

**Reply by badereddineqodia** [reply]
@Abodolla Oh, that's fantastic! Can you share with us what you have done? It would be amazing for the community!

**Reply by badereddineqodia** [reply]
@Abodolla any update ?

**Reply by sarahmanning190** [reply]
> I’ve done it successfully!

How were you able to integrate the Whatsapp embedded sign up?

**Reply by Abodolla** [reply]
@sarahmanning190 By some modification.

**Reply by sarahmanning190** [reply]
> @sarahmanning190 By some modification.

via Evolution API?

**Reply by muhsin-k** [reply]
@Abodolla This is a nice addition to have. We welcome the PR and are happy to test it.

**Comment by badereddineqodia** [comment]
any update ?

**Comment by ldaniel-jmz** [comment]
+1

**Comment by cakrumen** [comment]
+1

**Comment by muhsin-k** [comment]
We have started working on it. You can track the progress here https://github.com/chatwoot/chatwoot/pull/11612
